android/src/main/java/com/usabilla/reactnative/ubform/UsabillaBridge.java

```diff
@@ -55,12 +72,24 @@ public void onReceive(Context context, Intent intent) {
                 if (fragment != null) {
                     supportFragmentManager.beginTransaction().remove(fragment).commit();
                 }
+                emitReactEvent(getReactApplicationContext(), "UBFormDidClose", result);
                 return;
             }
+            final WritableMap resultError = Arguments.createMap();
+            resultError.putString(KEY_ERROR_MSG, "The form could not be shown because the activity doesn't extend from FragmentActivity");
+            emitReactEvent(getReactApplicationContext(), "UBFormLoadingFailed", resultError);
```

Comment:
This is not right.
We are in a receiver that listens for when we broadcast an event in the system for the form to be closed.
We check if the activity we are currently in is of type `FragmentActivity` and if not we cannot invoke the right entity to remove the form from view.

Therefore the message and the ReactEvent we emit are wrong.
I would say either we find a new ReactEvent to send around, or we simply log the information to console, also because this is a very unlikely (if not impossible) scenario given that the activity needs to already be of type `FragmentActivity` for the form to be loaded in the first place

Reply:
Done